source/macros.tex

```diff
@@ -156,6 +156,7 @@
 
 % Code and definitions embedded in text.
 \newcommand{\tcode}[1]{\CodeStylex{#1}}
+\newcommand{\mcode}[1]{\text{\tcode{#1}}}   % \tcode in math mode
```

Comment:
The `\text` command works in non-math mode as well. You could simply adjust the definition of `\tcode`:

    \newcommand{\tcode}[1]{\text{\CodeStylex{#1}}}

and it would work just fine in both math mode and text mode.

The only potential downside to using `\text` in text mode, it becomes the equivalent to `\mbox`. The contents of `\mbox` won't be split across lines. If that's a problem, we can further adjust the definition to work around this:

```latex
\newcommand{\tcode}[1]{%
  \ifmmode\text{\CodeStylex{#1}}\else\CodeStylex{#1}\fi
}
```

The benefit to this approach is that it reduces the cognitive load on the author. The author can use a single macro instead of having to decide between two different macros depending on the mode they're in.